src/actions/Entities/VaccinePrescriptionActions.js

```diff
@@ -219,9 +219,9 @@ const createVaccinationNameNote = (
   UIDatabase.write(() => UIDatabase.create('NameNote', newNameNote));
 };
 
-const getPcdEventID = (patientEventType, patientId) => {
+const getPcdNameNoteID = patientId => {
```

Comment:
Renamed because the old name was a little misleading (there is also a PatientEvent with code 'PCD' with an ID but this is not what the function is returning)